src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -489,6 +489,7 @@ private function addRouterSection(ArrayNodeDefinition $rootNode)
                                 ->scalarNode('host')->defaultValue('%router.request_context.host%')->end()
                                 ->scalarNode('scheme')->defaultValue('%router.request_context.scheme%')->end()
                                 ->scalarNode('base_url')->defaultValue('%router.request_context.base_url%')->end()
+                                ->scalarNode('url')->defaultValue('%router.request_context.url%')->end()
```

Comment:
Should we add a config validation to check if this or others are filled but not both? To avoid « conflict »?

Also what about deprecating others if it does the same?

Reply:
It's a good question. 

In this PR url syntax will override "standard" one. Validation may be useful too.
I think, that deprication is not needed, becouse old syntax can be useful. For example, user can pass only base_url.